suite-native/state/src/store.ts

```diff
@@ -25,6 +26,8 @@ const enhancers: Array<StoreEnhancer<any, any>> = [];
 
 if (__DEV__) {
     enhancers.push(devToolsEnhancer({ maxAge: 150 })!);
+    // logger
+    // middlewares.push(logger);
```

Comment:
remove this please

Reply:
I am thinking about keeping it because it's quite useful since redux devtools doesn't work

Reply:
What do you wanna keep? Commented out code doesn't belong to VCS from my PoV.

Reply:
Do we know why does not it work now?

Reply:
No idea, tried to debug it but no luck.

@yanascz I think this is fine, it's better than googling everytime you need to add debugger. But maybe I will hide this under some true/false const instead of commenting out.

Reply:
reworked [61b3b75](https://github.com/trezor/trezor-suite/pull/15184/commits/61b3b75ec0d7f238f34d1faada1ec2054303a0a6)